C++/test/algorithm/dynamic_programming/longest_decreasing_subsequence.cpp

```diff
@@ -0,0 +1,15 @@
+#define LONGEST_DECREASING_SUBSEQUENCE
+
+#include "third_party/catch.hpp"
+#include "include/algorithms/dynamic_programming/longest_decreasing_subsequence.hpp"
+
+TEST_CASE("Base cases", "[longest_decreasing_subsequence]") {
+	REQUIRE(longest_decreasing_subsequence(std::vector<int>({}) == 0);
+	REQUIRE(longest_decreasing_subsequence(std::vector<int>({1}) == 1);
+}
+
+TEST_CASE("Test cases", "[longest_decreasing_subsequence]") {
```

Comment:
Let's add a test case for when you have a list with multiple numbers but no decreasing sequence, i.e., 1, 2, 3, 4, 5, 6, 7, 8, 9.